test/nodeMock/ccd/ccdCaseConfig/CCDCaseField.js

```diff
@@ -26,7 +26,7 @@ const ccdFieldTemplate = {
 
     ],
     "complexACLs": [],
-    "display_context": "OPTIONAL",
```

Comment:
@paul-graham - do you know if it is intentional to set `display_context` to `null`?

Reply:
I _believe_ so. It looks like a deliberate change anyway. 🤷‍♂️

Reply:
And the tests all pass with that in place.

Reply:
👍 OK - must be intentional, then.